spec/support/authentication.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+def authentication_headers_for(user)
+  {
+    Authentication::REMOTE_USER_HEADER => user.email_address,
+    Authentication::NAME_HEADER => user.name,
+    Authentication::FIRST_NAME_HEADER => user.first_name
+  }
+end
```

Comment:
Worth considering a structure like https://github.com/sul-dlss/happy-heron/blob/main/spec/support/session_helpers.rb so that this method is only in the global namespace during request specs where it's needed, or do you think we'll want this in other sorts of specs?

Reply:
Will probably want it in at least request and system. In this case, seems like unnecessary complexity that might cause future confusion.